_rules/SC4-1-1+SC4-1-2-aria-state-or-property-allowed.md

```diff
@@ -27,7 +27,7 @@ The attribute is either an [inherited](https://www.w3.org/TR/wai-aria/#inherited
 
 ## Assumptions
 
-*There are currently no assumptions*
+- The ARIA `state` or `property` is being used to comply to WCAG.
```

Comment:
Thanks for the suggestion Wilco. This doesn't solve the entire problem though I think. Even if a state or property provides new information that isn't expressed trough ARIA in any other way it could still pass WCAG if the attribute is used incorrectly.

Say for example that the aria-multiline attribute is used incorrectly and the user is not told it is a multiline input field. What WCAG SC would this fail?

Reply:
> Say for example that the aria-multiline attribute is used incorrectly and the user is not told it is a multiline input field. What WCAG SC would this fail?

If the box is visually styled as multiline, but the markup doesn't indicate the box is multiline isn't this a failure of 1.3.1?